.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify that your changes did not broke the project. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
+
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
```

Comment:
I think these aren't relevant for our case, as we rely on Python installation with no AOT compiler toolchain needed. I would rather, if wanted, add Python/tooling versions:

```suggestion
**Test Configuration**:

* Python version:
* Poetry version:
```